cmd/dfget/app/server.go

```diff
@@ -18,13 +18,14 @@ package app
 
 import (
 	"fmt"
+	"os"
 	"path"
 
+	"github.com/spf13/cobra"
+
 	"github.com/dragonflyoss/Dragonfly/common/dflog"
 	"github.com/dragonflyoss/Dragonfly/dfget/config"
 	"github.com/dragonflyoss/Dragonfly/dfget/core/uploader"
-
-	"github.com/spf13/cobra"
```

Comment:
Please move "github.com/spf13/cobra" to its previous position.

Reply:
Done